invenio_rdm_records/resources/serializers/schemaorg/schema.py

```diff
@@ -0,0 +1,244 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 CERN.
+# Copyright (C) 2021 Northwestern University.
+# Copyright (C) 2023 Graz University of Technology.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Schemaorg based Schema for Invenio RDM Records."""
+from typing import Optional
+from edtf import parse_edtf
+from edtf.parser.grammar import ParseException
+from flask_resources.serializers import BaseSerializerSchema
+from pydash import py_
+import pycountry
+from idutils import normalize_orcid, normalize_ror
+from commonmeta import (
```

Comment:
same question as for `pydash`, ping @zzacharo

Reply:
`pycountry` is used elsewhere in InvenioRDM (for languages supported). `commonmeta` is a library I wrote specifically to help with metadata conversions.